From cdd2606b391155ff5704595dbc9726d073ca8c8f Mon Sep 17 00:00:00 2001
From: fundamental <mark.d.mccurry@gmail.com>
Date: Wed, 2 Mar 2016 21:13:20 -0500
Subject: [PATCH 0873/1936] Disable UI Title Rewriting

This feature is buggy and until it receives the attention
it needs, it will be (mostly) disabled.
---
 src/UI/Fl_Osc_Pane.cpp | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/UI/Fl_Osc_Pane.cpp b/src/UI/Fl_Osc_Pane.cpp
index 502c413f..49415163 100644
--- a/src/UI/Fl_Osc_Pane.cpp
+++ b/src/UI/Fl_Osc_Pane.cpp
@@ -16,6 +16,7 @@ void Fl_Osc_Window::init(Fl_Osc_Interface *osc_, std::string loc_)
 {
     title_ext  = new Osc_DataModel(osc_);
     title_ext->doUpdate("/ui/title");
+#if 0
     title_ext->callback = [this](string next) {
         rewrite_rule = next;
         //printf("old: %s\n", title_orig.c_str());
@@ -35,6 +36,9 @@ void Fl_Osc_Window::init(Fl_Osc_Interface *osc_, std::string loc_)
         //title_new = title_orig + next;
         this->label(title_new.c_str());
     };
+#else
+    title_ext->callback = [this](string next) {};
+#endif
     title_orig = label();
 
     osc   = osc_;
-- 
2.47.0

